src/enums/reviewSummaryFlags.ts

```diff
@@ -5,8 +5,8 @@
  * */
 export enum ReviewSummaryFlags {
   'effective-date-time',
-  'court-order',
+  'document-delivery-section',
```

Comment:
Could you update the comment on line 4 to say that order should follow the component layout, please?

Reply:
Absolutely

Reply:
Fixed in next commit